tests/baselines/reference/jsdocFunctionType.symbols

```diff
@@ -61,3 +61,9 @@ z.length;
 >z : Symbol(z, Decl(functions.js, 26, 3))
 >length : Symbol(length, Decl(functions.js, 12, 27))
 
+/** @type {function ("a" | "b"): 1 | 2} */

```

Comment:
This doesn't actually test the ability to parse a number literal as a parameter type, that would be `function("a" | "b", 1 | 2): whatever`.